impeller/tessellator/tessellator.cc

```diff
@@ -218,6 +231,43 @@ Tessellator::Result Tessellator::Tessellate(
   return Result::kSuccess;
 }
 
+std::pair<std::vector<Point>, std::vector<uint16_t>>
+Tessellator::TessellateConvex(const Path& path, Scalar tolerance) {
+  std::vector<Point> output;
+  std::vector<uint16_t> indices;
+
+  point_buffer_->clear();
+  auto polyline =
+      path.CreatePolyline(tolerance, std::move(point_buffer_),
+                          [&](Path::Polyline::PointBufferPointer point_buffer) {
```

Comment:
The lambda capture can be reduced to `[this]`

Reply:
Done